### DISCUSSION TITLE
购买了一年专业版，无法授权其他设备

### DISCUSSION THREAD

**Discussion by Homeoflife** [getlantern/lantern-forum-cn#907] [Q&A]
第1个问题，2024年9月24日付费购买了一年专业版。在我的电脑里面打开蓝灯，点击右上角头像， 没有出现“授权其他设备使用专业版账号”的选项。只出现了“授权设备”选项。我现在需要授权同事的电脑，请问怎么解决？
<img width="843" alt="缺少“授权其他设备使用”选项" src="https://github.com/user-attachments/assets/d59c3e49-bf85-43d6-8153-004d871e2b5c">

第2个问题，在电脑上点击下载Lantern安卓版，出现二维码， 但是用手机微信扫描此二维码之后，出现提示“已停止访问该网页”。华为和苹果手机扫描出来都是同样结果。我们需要在手机上使用Lantern。请问怎么解决？

**Comment by MaxMKMax** [comment]
@Homeoflife 您好. 我帮您问问看怎么解决. 请问您是window 7? Window 7 我们就支持不了了.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by Homeoflife** [reply]
Mike您好。我的电脑是Windows 11的。请问我需要怎么操作，才能授权其他设备使用我的专业版蓝灯账号？
第2个问题，在电脑上点击下载Lantern安卓版，出现二维码， 但是用手机微信扫描此二维码之后，出现提示“已停止访问该网页”。华为和苹果手机扫描出来都是同样结果。我们需要在手机上使用Lantern。请问怎么解决？
